install_scripts/templates/kubernetes/init.sh

```diff
@@ -166,6 +163,21 @@ initKube() {
     logSuccess "Kubernetes Master Initialized"
 }
 
+# workaround for https://github.com/kubernetes/kubeadm/issues/998
+patchCoreDNS() {
+    n=0
+    while ! KUBECONFIG=/etc/kukbernetes/admin.conf kubectl -n kube-system get deployment coredns &>/dev/null; do
+        n="$(( $n + 1 ))"
+        if [ "$n" -ge "120" ]; then
+            break
```

Comment:
is your intention just to let the next line fail?

Reply:
Yes